src/mgr/ActivePyModules.cc

```diff
@@ -418,6 +425,29 @@ void ActivePyModules::notify_all(const LogEntry &log_entry)
   }
 }
 
+bool ActivePyModules::get_store(const std::string &module_name,
+    const std::string &key, std::string *val) const
+{
+  PyThreadState *tstate = PyEval_SaveThread();
+  Mutex::Locker l(lock);
+  PyEval_RestoreThread(tstate);
+
+  const std::string global_key = PyModuleRegistry::config_prefix
+    + module_name + "/" + key;
+
+  dout(4) << __func__ << "key: " << global_key << dendl;
+
+
+  Mutex::Locker lock(config_cache.lock);
+
+  if (config_cache.config.count(global_key)) {
+    *val = config_cache.config.at(global_key);
```

Comment:
generally speaking better to do
```
auto p = config_cache.config.find(global_key);
if (p != config_cache.config.end()) {
  *val = p->second;
  return true;
} else {
  return false;
}
```
to avoid a double lookup